test/unit/ngsiv2/lazyAndCommands/lazy-devices-test.js

```diff
@@ -343,7 +360,7 @@ describe('IoT Agent Lazy Devices', function() {
 
         it('should return the empty value', function(done) {
             request(options, function(error, response, body) {
-                var entities = JSON.parse(body);
+                var entities = body;
```

Comment:
This is weird...

This should be working even with the implementation based in GET /v2/entities (as the response of that request and POST /v2/op/query are the same). However, I have needed to modify this way in order the test to pass.